yarn-project/pxe/src/contract_function_simulator/pxe_oracle_interface.ts

```diff
@@ -336,80 +337,9 @@ export class PXEOracleInterface implements ExecutionDataProvider {
     }));
   }
 
-  public async syncTaggedLogsAsSender(
```

Comment:
This function was dropped and re-implemented in `sync_sender_tagging_indexes.ts` file. The new function doesn't share any code with this.